.agent/skills/moderation-email/md2html.py

```
@@ -0,0 +1,158 @@
+        '<div style="font-family: -apple-system, Arial, sans-serif; '
+        'font-size: 14px; line-height: 1.55; max-width: 52em; '
+        'margin: 2em auto; padding: 0 1em;">',
+        '<style>table{border-collapse:collapse;margin:0.8em 0}'
+        'td,th{border:1px solid #ccc;padding:4px 9px;text-align:left;'
+        'vertical-align:top}th{background:#f2f2f2}'
+        'code{background:#f4f4f4;padding:1px 4px;border-radius:3px}</style>',
+    ]
+    lines = src.split('\n')
+    if lines and lines[0].startswith(':::'):
+        mode = lines[0][3:].strip()
+        end = next(i for i, l in enumerate(lines[1:], 1) if l.strip() == ':::')
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)